gravityflow/gravityflowpdf
# Update mPDF to v7.1

re: [HS#7984](https://secure.helpscout.net/conversation/775618356/7984?folderId=1113535)

- Removes mPDF from the includes directory
- Adds the includes/fonts directory
- Uses composer to include mPDF v7.1
- Defines the minimum PHP requirements which the add-on framework will handle checking and displaying the appropriate admin notice
  + v5.6
  + curl, gd, and mbstring extensions
  + mb_substr and mb_regex_encoding functions
- Updates the mPDF initialization to define the location of the fonts and tmp directories
- Adds the gravityflowpdf_mpdf_config filter so the initialization properties can be changed

## Testing Instructions
- Test as many forms with PDF feeds as you can to ensure the PDFs are created as expected and no notices/warnings occur, ensure tests include templates using custom HTML, various fonts and scripts (i.e Thai).
- Try deleting the wp-content/uploads/gravityflowpdf/tmp directory and then test another form to ensure the directory is recreated during PDF creation
- Check that PDFs are not created when running on PHP 5.5 or older
